tests/Benchmarks/Azure/CosmosDb.cs

```diff
@@ -35,26 +35,26 @@ public class CosmosDb
     static Sha256 sha;
 
     [GlobalSetup]
-    public void CosmosDbBench()
+    public void Setup()
```

Comment:
As part of this clean up change, can you add a System.Azure.Experimental.Benchmarks namespace and move this test to a System.Azure.Experimental directory under Benchmarks?